.github/workflows/renovate.yml

```diff
@@ -0,0 +1,23 @@
+name: Renovate
+on:
+  schedule:
+    # The "*" (#42, asterisk) character has special semantics in YAML, so this
+    # string has to be quoted.
+    # https://crontab.guru/#54_6-18/4_*_*_MON-THU : Run every 4 hours
+    - cron: "54 6-18/4 * * MON-THU"
```

Comment:
What was the decision behind running this every 4 hours? Could once a day be frequent enough?

Reply:
Yes, once a day is probably enough, but I was curious about how good automerge is. If this works as well as advertised, we should get less noise than with dependabot.

Reply:
@frankie297 – Now that this is actually controlled by the renovate bot, we can actually simply use the configuration [as per the docs](https://docs.renovatebot.com/presets-schedule/).

```json
{
  "schedule": [
    "every weekday"
  ]
}
```